pyodata/v2/service.py

```diff
@@ -616,6 +620,11 @@ def count(self, inline=False):
             self._count = True
         return self
 
+    def next_url(self, next_url):
+        """Sets the URL to be fetched. Takes precedence over all query parameters."""
```

Comment:
Link to odata spec pls

Reply:
Hi @phanak-sap,

This is also the same issue what we see for SAP SuccessFactors.

Sharing the documentation for future -
https://help.sap.com/viewer/d599f15995d348a1b45ba5603e2aba9b/2111/en-US/93ef8631b93b4d58be235b047dae2b57.html

Reply:
@Vinuthan I like the wording in the referenced documentation "Client-Side Pagination" vs "Server-Side Pagination"